composer.json

```diff
@@ -1 +1,42 @@
-{}
\ No newline at end of file
+{
+    "name": "altapay/altapay-for-woocommerce",
+    "license": "MIT",
+    "description": "AltaPay: Payments less complicated",
+    "keywords": [
+        "payment",
+        "altapay"
+    ],
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "config": {
+        "preferred-install": "dist",
+        "sort-packages": true,
+        "bin-dir": "bin"
+    },
+    "archive": {
+        "exclude": [
+            "phpstan.neon.dist",
+            "phpunit.xml.dist"
+        ]
+    },
+    "require": {
+        "php": "^5.6 || ^7.0",
+        "guzzlehttp/psr7": "^1.4",
+        "guzzlehttp/guzzle": "^6.0",
+        "symfony/options-resolver": "~2.6 || ~3.0 || ^4.0",
+        "altapay/api-php":"^3.0",
+        "psr/log": "^1.1",
+        "symfony/dependency-injection": "^5.2",
+        "psr/event-dispatcher": "^1.0",
+        "laminas/laminas-httphandlerrunner": "^1.3",
```

Comment:
These also appear to be dependencies of dependencies and not used directly by the project.